src/CommunityToolkit/Xamarin.CommunityToolkit/Views/RangeSlider/RangeSlider.shared.cs

```diff
@@ -59,7 +60,7 @@ public static BindableProperty UpperThumbSizeProperty
 			= BindableProperty.Create(nameof(UpperThumbSize), typeof(double), typeof(RangeSlider), -1.0, propertyChanged: OnLayoutPropertyChanged);
 
 		public static BindableProperty TrackSizeProperty
-			= BindableProperty.Create(nameof(TrackSize), typeof(double), typeof(RangeSlider), 3.0, propertyChanged: OnLayoutPropertyChanged);
+			= BindableProperty.Create(nameof(TrackSize), typeof(double), typeof(RangeSlider), 4.0, propertyChanged: OnLayoutPropertyChanged);
```

Comment:
We could have like a RangeSlider constants where define the default ThumbSize, TrackSize, etc.

Reply:
Yeah, that's a nice point.
But if it isn't critical, let's skip it for now